android/java/res/layout/brave_talk_opt_in_layout.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2021 The Brave Authors. All rights reserved.
+     Use of this source code is governed by a BSD-style license that can be
+     found in the LICENSE file. -->
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
We should avoid using relativelayout. we can use Framelayout here.

Reply:
Fixed